insights/specs/default.py

```diff
@@ -190,7 +190,7 @@ class DefaultSpecs(Specs):
     ethtool_c = foreach_execute(ethernet.interfaces, "/sbin/ethtool -c %s")
     ethtool_g = foreach_execute(ethernet.interfaces, "/sbin/ethtool -g %s")
     ethtool_i = foreach_execute(ethernet.interfaces, "/sbin/ethtool -i %s")
-    ethtool_k = foreach_execute(ethernet.interfaces, "/sbin/ethtool -k %s")
+    ethtool_k = foreach_execute(ethernet.interfaces, "/bin/ethtool -k %s")
```

Comment:
@TZ3070 - Is this change required?

Reply:
No, it is a mistake.

Reply:
@TZ3070 - This update looks good to me. But as it's also a kind of SPEC change to the core collection, please raise an issue (not MR) to the insights-core-assets repo for this change.

Reply:
@TZ3070 Can you please verify is the path is `/bin/teamdctl`? On my rhel8.5 system it shows `/usr/bin/teamdctl`

Reply:
@psachin 
After checking, The path is '/usr/bin/teamdctl'. '/bin/teamdctl' is symbolic link.